index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <!-- <meta name="viewport" content="width= , initial-scale=1.0"> -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
- [ ] Kindly 😊 try to remove the unnecessary comment here because this makes your codebase ugly ,so we want your code to look nice now and in the next projects try to not put unnecessary comments at the base code to make the code organized and nice for the others😃